Settings.cs

```diff
@@ -95,114 +97,184 @@ internal static Font FontCapsuleParameterData
         }
         #endregion
 
+        // Cache frequently-used values to improve performance
```

Comment:
GH_SettingsServer uses a SortedList so it has a quick read. I don't think it's a noticeable improvement and that's why I didn't optimise it. But since you did, I would add it if it was in a different pull request (I can't merge it because the changes in MenuItem.cs stop working for clusters).

Reply:
This might not be a huge improve as I haven't benchmarked it. But SortedList isn't quite very efficient. It looks up in all GH settings, do boxing/unboxing (due to it not being a generic type), etc.